app/views/locations/_directory_filter.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= form_tag locations_path, method: 'get', class: 'l-filter-form' do %>
+  <ul class="pagination-atoz">
+    <% (?A..?Z).each do |letter| %>
+      <li class="pagination-atoz__letter t-pagination__letter">
+        <%=
+          radio_button_tag(
+              :letter,
+              letter,
+              letter == @sorting_params[:currentField],
+              disabled: letter != @sorting_params[:currentField] &&
+                        !@sorting_params[:availableLetters].include?(letter),
+              class: 'pagination-atoz__checkbox js-submit-form'
+          )
+        %>
+        <%= label_tag :"letter_#{letter}", letter, class: 'pagination-atoz__label' %>
+      </li>
+    <% end %>
+  </ul>
+
+  <div>
+    Display:
```

Comment:
To me it would make sense if the `Display:` was the text for the `<%= label_tag %>` below